package.json

```diff
@@ -19,12 +19,12 @@
     "test": "mocha --recursive tests/"
   },
   "engines": {
-    "node": ">=6.10.0"
+    "node": ">=10.0.0"
```

Comment:
I coupled in a bump of the min supported version -- 10 is now the lowest LTS version node offers (and gives us async/await).